### ISSUE TITLE
Request: Sell Upgrades

### ISSUE THREAD

**Issue by tdk1007** [grahamgilchrist/hotac-squad-builder#136]
Some groups allow selling back upgrade cards for half their value. Don't think this is mentioned in the official rules, but it would be a nice feature.

**Comment by grahamgilchrist** [comment]
There have been a number of requests for additional ruleset/house rules type functionality.

I don't think this one would be too hard to implement technically, but at the moment I'm sticking with core HOTAC rules only. I need to think if there is even a way to allow custom rules which can be optional for some users. Part of the remit of this app was to guide my play group on what they can/can't choose, and adding "optional" rulesets confuses that somewhat.

It might be a bit more of a challenge to squeeze more into the interface, especially on mobile. Potentially it would fit next to the "lose upgrade" button, and would essentially work the same way, just with gaining some XP at the same time.

**Comment by tdk1007** [comment]
I didn't realize the "lose upgrade" button was added in. That works great too!

**Comment by grahamgilchrist** [comment]
Yes, you can get the equivalent of this functionality by "losing" the upgrade and then just adding some extra XP for the sale. It just won't be obvious in the history where the XP came from
